.gitignore

```diff
@@ -10,4 +10,5 @@
 **/*.log
 version.txt
 landingzones
-public/
\ No newline at end of file
+public/
+.data
```

Comment:
what is the .data dir from?

Reply:
It's where I put the symphony yaml while testing. I added to gitignore so I don't accidentally check in the test symphony yaml file.